### PR TITLE
Make search bar compatible with Vuetify

### PR DESCRIPTION
**Pull request by keelywan**
## Search Bar 
The search bar is now made with Vuetify components. 

vuetify-search

## Error
The search bar will also display an error message if both input fields are empty when the search button is clicked.
error-message

### CODE REVIEW

**Review by Aradhya-B — APPROVED** [review]
Man I love Vuetify! Great work!

File: src/main/webapp/components/SearchBar.vue [hunk 1]
```diff
@@ -43,17 +39,19 @@ module.exports = {
       }
       else /* Form validated */ {
         this.error = "";
-        this.queryDatabase(this.school, this.course)
-          .then(result => this.$emit('searchresult', result))
-          .catch(error => this.error = error);
+        this.queryDatabase(this.school, this.course);
```
Comment (andres-holguin) [inline src/main/webapp/components/SearchBar.vue]: Should we handle the error that `queryDatabase()` can throw in some way?

**Review by andres-holguin — APPROVED** [review]
Small error handling question, but LGTM otherwise :)

File: src/main/webapp/components/SearchBar.vue (hunk 1 above)
Comment (keelywan) [reply src/main/webapp/components/SearchBar.vue]: Sure I can add that

**Review by cftorres — APPROVED** [review]
All of these Vueify changes look great!

**Pull request merged**